Webapp/sources/blueprints/save_reaction/routes.py

```diff
@@ -394,6 +394,61 @@ def autosave() -> Response:
     return jsonify({"feedback": feedback})
 
 
+@save_reaction_bp.route("/clone_reaction", methods=["POST", "GET"])
+@login_required
+def clone_reaction() -> Response:
+    """
+    Takes reactions data from previously saved reaction and copies it into a new reaction which is saved to the database
+    real_product_mass and unreacted_reactant_mass are removed from the reaction data for users to input upon reaction completion
+
+    Returns:
+        Response as JSON, either New Reaction Made or Reaction Name is not Unique
+
+    """
+
+    old_reaction = services.reaction.get_current_from_request_form()
+
+    new_reaction_name = request.form['reactionName']
+    workbook_name = request.form["workbook"]
+    workgroup_name = request.form["workgroup"]
+    new_reaction_id = request.form['newReactionID']
```

Comment:
Similar story as above, use `request.form.get` and check for `None`. If `None` return a 400 Bad Request response.